zkevm-circuits/src/tx_circuit.rs

```diff
@@ -671,19 +724,223 @@ impl<F: Field> SubCircuitConfig<F> for TxCircuitConfig<F> {
         });
 
         ///////////////////////////////////////////////////////////////////////
-        ///////////////  constraints on BlockNum  /////////////////////////////
+        ///////////////  constraints on num_all_txs  // ///////////////////////
+        ///////////////////////////////////////////////////////////////////////
+        meta.create_gate("tx_nonce", |meta| {
```

Comment:
```suggestion
        meta.create_gate("copy tx_nonce", |meta| {
```

Reply:
fixed in https://github.com/scroll-tech/zkevm-circuits/pull/764/commits/bfd23e461fb503ff5fd8a74867829225a6e57fc8